lighthouse-core/config/default.json

```diff
@@ -3,6 +3,7 @@
     "passName": "defaultPass",
     "recordNetwork": true,
     "recordTrace": true,
+    "pauseAfterTraceMs": 500,
```

Comment:
Yeah true, just was very similar in spirit to the pauseAfterLoadMs so wanted to be consistent. How about `pauseBeforeTraceEnd`? Nix the `Ms` either way?